Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2144,12 +2215,12 @@ extension BrowserViewController: TabsBarViewControllerDelegate {
 }
 
 extension BrowserViewController: TabDelegate {
-
   func tab(_ tab: Tab, didCreateWebView webView: WKWebView) {
+    webView.scrollView.contentInsetAdjustmentBehavior = .never
     webView.frame = webViewContainer.frame
+    
     // Observers that live as long as the tab. Make sure these are all cleared in willDeleteWebView below!
     KVOs.forEach { webView.addObserver(self, forKeyPath: $0.rawValue, options: .new, context: nil) }
-    webView.scrollView.addObserver(self.scrollController, forKeyPath: KVOConstants.contentSize.rawValue, options: .new, context: nil)
```

Comment:
We can remove `KVOConstants.contentSize` enum value if not used anymore

Reply:
Im gonna remove this anyways but your comment here made me question the safety of just relying on the zoom to show the toolbars if they're not longer collapsable after a web page changes sizes, so I've changed the code to observe `contentSize` instead of relying on `scrollViewDidZoom`